frontend/.eslintrc.js

```diff
@@ -8,6 +8,7 @@ const sharedRules = {
   "@typescript-eslint/no-explicit-any": "off",
   "react-hooks/exhaustive-deps": "off", // TODO: remove this line
```

Comment:
Can we go ahead and turn the react-hooks rules back on (by removing lines 9-10)?